contribs/gmf/src/services/themesservice.js

```diff
@@ -392,6 +401,9 @@ gmf.Themes.prototype.loadThemes = function(opt_roleId) {
 
   goog.asserts.assert(this.treeUrl_, 'gmfTreeUrl should be defined.');
 
+  this.deferred_ = this.$q_.defer();
+  this.promise_ = this.deferred_.promise;
```

Comment:
Not sure it will work if we have:
```
gmfThemes.getThemesObject().then(function(themes) {
  //
});
gmfThemes.loadThemes();
```

The promise used in `getThemesObject` is removed in `loadThemes` and the function is never called.